services/status/status.proto

```diff
@@ -0,0 +1,11 @@
+message Request{
+
```

Comment:
Can we later extend an empty protobuf packet ? Maybe later we'd want to have a "status request" to exactly define what info we want or something in the same spirit. Just asking it's not a big deal.